src/pages/workspace/WorkspacePeoplePage.js

```diff
@@ -0,0 +1,328 @@
+import React, { Fragment } from 'react';
+import {
+    View, FlatList, ScrollView,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import styles from '../../styles/styles';
+import ONYXKEYS from '../../ONYXKEYS';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Navigation from '../../libs/Navigation/Navigation';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import compose from '../../libs/compose';
+import {flatListRef} from '../../libs/ReportScrollManager';
+import {removeMembers} from '../../libs/actions/Policy';
+import Avatar from '../../components/Avatar';
+import Button from '../../components/Button';
+import Checkbox from '../../components/Checkbox';
+import Text from '../../components/Text';
+import ROUTES from '../../ROUTES';
+import ConfirmModal from '../../components/ConfirmModal';
+import personalDetailsPropType from '../personalDetailsPropType';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../../components/withWindowDimensions';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+
+    ...windowDimensionsPropTypes,
+
+    /** The personal details of the person who is logged in */
+    personalDetails: personalDetailsPropType.isRequired,
+
+    /** The policy passed via the route */
+    policy: PropTypes.shape({
+        /** The policy name */
+        name: PropTypes.string,
+    }),
+
+    /** URL Route params */
+    route: PropTypes.shape({
+        /** Params from the URL path */
+        params: PropTypes.shape({
+            /** policyID passed via route: /workspace/:policyID/people */
+            policyID: PropTypes.string,
+        }),
+    }).isRequired,
+};
+
+const defaultProps = {
+    policy: {
+        name: '',
+    },
+};
+
+class WorkspacePeoplePage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            selectedEmployees: [],
+            isRemoveMembersConfirmModalVisible: false,
+        };
+
+        this.inviteUser = this.inviteUser.bind(this);
+        this.removeUsers = this.removeUsers.bind(this);
+        this.toggleUser = this.toggleUser.bind(this);
```

Comment:
This does not need to be bound in the constructor since all usages are inside anonymous arrow functions whose scope is bound to the component already.

Reply:
Very well spotted, thank you! Had to update how I call the functions too.